_overviews/getting-started/scala-ides.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: singlepage-overview
+title: Scala IDEs
+
+partof: scala-ides
+
+permalink: /tutorials/:title.html
+
+keywords:
+- Scala
+- IDE
+- JetBrains
+- IntelliJ
+- VSCode
+- Metals
+---
+
+By Maciej Gorywoda
```

Comment:
Although we very much appreciate you providing a first version of the page, I suggest removing the author credit. We don't generally include them on short factual pages; I think the text is likely to evolve a lot over time; and we have found that having an author credit discourages edits.